readme.md

```diff
@@ -26,6 +32,14 @@ To install Administrator as a Composer package to be used with Laravel 4, simply
 )
 ```
 
+### Laravel 5
+
+Then publish the config file with `php artisan vendor:publish`. This will add the file `app/config/administrator.php`. This [config file](http://administrator.frozennode.com/docs/configuration) is the primary way you interact with Administrator.
+
+Next to that all the assets, views and translation files will be published too.
+
+### Laravel 4
+
```

Comment:
We can remove the Form facade altogether. Ignoring the obvious refactoring that needs to happen here, a quick solution is:

`php
app('request')->session()->token();
`